KMaina/Fast-Food-App
# #160748981 Redesign the Admin Orders Page

#### What does this PR do?

Allows an admin to edit and order

#### Description of Task to be completed?

The admin should click on the edit button to redirect to the order edit page

#### How should this be manually tested?

- Clone the repo onto your local machine.
- Open a terminal and run cd Fast-Food-App
- run `git clone https://github.com/KMaina/Fast-Food-App.git`
- run `git checkout ch-redesign-admin-orders-160748981`
- In the folder where you cloned the repo, you should open the UI folder.
- Open the admin-orders.html to see it in your browser for review.

#### Any background context you want to provide?

This is a redesign of the page
#### What are the relevant pivotal tracker stories?

#160748981
